system/Database/MigrationRunner.php

```diff
@@ -956,8 +956,9 @@ public function ensureTable()
 				'null'       => false,
 			],
 			'class'     => [
-				'type' => 'TEXT',
-				'null' => false,
+				'type'       => 'VARCHAR',
+				'constraint' => 255,
+				'null'       => false,
```

Comment:
While I think this is a good change, are we absolutely sure this is backwards-compatible if anyone has built something on top of the migrations table?

Reply:
I don't know what happened but when I moved the Models tests out of Database Live, I get errors on MigrationRunnerTest::testGetHistory for Sqlsrv. It says that I cannot make equal to comparisons for TEXT. This is what I found to be the solution.

Reply:
Okay, it kind of baffles me. Let's see if anyone else has thoughts but I'm conditional okay with it. This had no place being a `TEXT` to being with.